CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+- Updated React packages to version 16.8.6
```

Comment:
```suggestion
- React packages to version 16.8.6
```

Reply:
it doesn't mean the same thing without the verb